src/components/button/button.stories.mdx

```diff
@@ -478,7 +479,7 @@ Buttons can be rendered with just an icon.
       <Button mt={2} destructive ml={2} iconType="bin" aria-label="Delete" />
       <Button
         mt={2}
-        buttonType="dashed"
+        disabled
```

Comment:
Hi @Parsium, since dashed buttons are going to be deleted anyway -  this change was to replace them with another type of button. In that case disabled. Also there are duplicates of props for button types.